template.json

```
@@ -10,6 +10,10 @@
             "bonuses": {
               "check": "",
               "save": ""
+            },
+            "minimum": {
+              "check":null,
```

Sorry to nitpick but these are missing a space after the colon:

```suggestion
              "check": null,
```